### ISSUE TITLE
AArch64 (ARM 64 bits) is not supported

### ISSUE THREAD

**Issue by ggardet** [aelve/charsetdetect-ae#7]
AArch64 is currently not supported, which leads to the following error:
```
[  234s] In file included from libcharsetdetect/nspr-emu/prcpucfg.h:38:0: error:
[  234s]     0,
[  234s]                      from libcharsetdetect/nspr-emu/prtypes.h:58,
[  234s]                      from libcharsetdetect/nscore.h:1,
[  234s]                      from libcharsetdetect/charsetdetectPriv.h:6,
[  234s]                      from libcharsetdetect/charsetdetect.cpp:3:
[  236s] 
[  236s] libcharsetdetect/nspr-emu/prcpucfg_linux.h:650:2: error:
[  236s]      error: #error "Unknown CPU architecture"
[  236s]      #error "Unknown CPU architecture"
[  236s]       ^~~~~
```
Could you add support to it, please?

**Comment by neongreen** [comment]
This seems to be a matter of doing https://github.com/batterseapower/libcharsetdetect/issues/3. I'll get to it.

**Comment by neongreen** [comment]
Fixed in 1.1.0.4.
